i18nExtractKeys.vite.js

```diff
@@ -0,0 +1,64 @@
+import path from "path";
+import fs from "fs";
+
+const uniqueKeys = new Set();
+
+function extractRegexPlugin({ extraKeys } = {}) {
```

Comment:
@jardakotesovec, I don't think we've yet considered a way for plugins to use this without duplicating it in the codebase; any recommendations?

Reply:
My conclusion is here - https://github.com/pkp/pkp-lib/issues/9392#issuecomment-2789492802   Opened to reconsider.